roles/mysql/vars/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+# vars file for mysql
+#
+#
+mysql_name: mariadb
```

Comment:
Values set in this file will override anything in `group_vars` - is that what we want? I suspect these should go in `defaults/main.yml`.